client/src/components/ButtonSubmit/ButtonSubmit.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.ButtonSubmit {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: 'Reem Kufi', sans-serif;
+}
+
+.ButtonSubmit-followed,
+.ButtonSubmit-favorited {
+  background: #FB836F;
+  border-radius: 4px;
+  width: 200px;
+  height: 50px;
+  color: #FAFAFA;
+  font-size: 18px;
+}
+
+.ButtonSubmit-next,
+.ButtonSubmit-finish {
+  width: 302px;
+  height: 65px;
+  background: #FAFAFA;
+  border-radius: 3px;
+  font-size: 36px;
+}
+
+.ButtonSubmit-register {
+  width: 302px;
+  height: 65px;
+  background: #6A4C93;
+  border-radius: 5px;
```

Comment:
Can you pick a single `border-radius` value and add that to the base button class?